src/lib/cms-content.ts

```diff
@@ -34,12 +35,12 @@ const itemMap = ({
   fileDownload = null,
 }: any): ContentItem => {
   return {
+    id,
```

Comment:
So my thought was to use the `slug` as a unique identifier. Just wanted to throw that out there and get your opinion on if you think that's a valid approach or if using the `id` adds more value.

Reply:
I just figured we might as well use `id` since its already there and it's a bit more obvious as its unique identifier. I was also planning on adding `id` to `Lessons` when it comes time to integrate the course page, so my thought was to keep it consistent